src/main/java/org/cryptomator/frontend/fuse/OpenFileFactory.java

```diff
@@ -67,8 +67,15 @@ public void close(long fileHandle) throws ClosedChannelException, IOException {
 		}
 	}
 
-	public int getOpenFileCount(){
-		return openFiles.size();
+	/**
+	 * Tests, if there exists dirty {@link OpenFile}s
+	 * This method is neither atomic regarding the set of open files nor regarding each open file individually. Therefore, external synchronization is required.
+	 *
+	 * @return {@code true} if and only if at least one dirty {@link OpenFile} exists. Otherwise {@code false}.
+	 * @see OpenFile#isDirty()
+	 */
+	boolean hastDirtyOpenFiles() {
```

Comment:
"has dirty files" sounds like adult content :sweat_smile:

```suggestion
	/**
	 * Tests, if any {@link OpenFile} is considered <em>dirty</em>, i.e. has pending changes.
	 * This method is neither atomic regarding the set of open files nor regarding each open file individually. Therefore, external synchronization is required.
	 *
	 * @return {@code true} if and only if at least one dirty {@link OpenFile} exists. Otherwise {@code false}.
	 * @see OpenFile#isDirty()
	 */
	boolean hasPendingChanges() {
```

Reply:
Changed in https://github.com/cryptomator/fuse-nio-adapter/pull/80/commits/842f5076d475eef72082fca51bc6862286fa1819:

I didn't come up with a better name, but would appreaciate if we stick to "hasDirty...", since it does not suggest anything and simply reflects the code.